docs/src/pages/[platform]/theming/style-attributes/examples/PositionStyles.tsx

```diff
@@ -0,0 +1,13 @@
+import { PropertyTable } from './PropertyTable';
+
+export const positionProperties = [
```

Comment:
We probably don't want to hardcode the properties, we can use the `Catalog` or `allTypesData` generated from https://github.com/aws-amplify/amplify-ui/pull/2090/

Reply:
Given our timeline, rather than refactoring now, I think we should have a follow up item to remove hardcoded maps.

Reply:
agreed this would be better pulled from another location, will follow upon this one